decision_making.md

```diff
@@ -0,0 +1,41 @@
+
+The various Jupyter decision making bodies often have vast scopes and members are often busy. As a result, there are typically a subset of members who have the time, interest, and expertise to dive into the details of a given decision. We believe that open, transparent, and time-bound voting with no quorum will enable these ad-hoc, informal subgroups to form spontaneously on a per-decision basis, in a manner that supports our goals of agility and participation. We have found that formal quorums slow decision making substantially, and don’t actually guarantee true participation.
+
+Once a decision has been made during the consensus seeking phase, or by a vote, the decision making body will record the decision in an appropriate location that is as open as possible given the decision and work of the particular body. 
+
+## Guidance for decision makers and those participating in decision making
+
+- We encourage teams to always seek to balance the core guiding principles of this process as stated above: balancing agile decision making with an open and collaborative culture of participation from all relevant stakeholders.
+- During the consensus seeking phase, don’t interpret a lack of participation, discussion, or feedback as consensus or the lack thereof. Dissent can be silent and sometimes people are supportive but busy. 
+- If you are interested in a decision being made, it is your responsibility to encourage voting member/stakeholders/community participation in the decision making process. If you can’t get such participation, you may want to hold off on doing any significant work on the matter.
+- We recognize that a no quorum voting scheme could be mistaken for a low participation voting scheme – that is not our goal. To guard against this, teams should always strive for broad participation.
+- Special guidelines for software projects making decisions:
+  - Add a link to this document in your issue and PR templates along with a link to the decision making body membership (see recommended language below).
+  - Create a “Consensus Seeking” and “Vote” tags to make it easy to find issues/PRs where a decision is being made.
+  - When a vote is called on GitHub, the sponsor should summarize the decision and paste a checklist of the voting members to use in voting in the top entry (description) of the issue/PR.
+  - When communications (in whatever number of channels) point to a discussion that requires a project-wide decision, and before major implementation work starts, a standalone issue should be opened that summarizes the issue, points to relevant prior discussions, and calls for an explicit vote/decision. This issue should be tightly scoped to the relevant decision only, and include the "Vote" tag.
+  - When an implementation of a decision appears in pull request, link to the relevant “Vote” issue.
+  - This decision making process is incompatible with individuals merging their own PRs in most situations.
```

Comment:
We can distinguish here between individuals reviewing their own PRs and merging their own PRs. In some systems, once other people have signed off on a PR, the original author or anyone else can merge it (for example, it might be signed off conditional on tests passing, and the original author might merge once tests have passed).

Reply:
Also this doesn't account for the somewhat common (at least not uncommon) occurrence of two people working on a PR, each reviewing the other's work, and then one of them merging it.

Reply:
We should remove this statement because each repository's contributors already have a strong culture of what is acceptable when it comes to merging pull requests.